bin/unconstrained_sandbox.jl

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env julia
+
+
+# require(joinpath(Pkg.dir("Celeste"), "src", "Util.jl"))
+# require(joinpath(Pkg.dir("Celeste"), "src", "CelesteTypes.jl"))
+# require(joinpath(Pkg.dir("Celeste"), "src", "ElboDeriv.jl"))
+# require(joinpath(Pkg.dir("Celeste"), "src", "OptimizeElbo.jl"))
+
+#include(joinpath(Pkg.dir("Celeste"), "src", "Transform.jl"))
+#include(joinpath(Pkg.dir("Celeste"), "src", "CelesteTypes.jl"))
+#include(joinpath(Pkg.dir("Celeste"), "src", "ElboDeriv.jl"))
+#include(joinpath(Pkg.dir("Celeste"), "src", "OptimizeElbo.jl"))
+
+
+#include(joinpath(Pkg.dir("Celeste"), "test", "test_derivs.jl"))
+
+using Celeste
+using CelesteTypes
+using Transform
+
+import OptimizeElbo
+import ElboDeriv
+
+using Base.Test
+using SampleData
```

Comment:
I do `using CelesteTypes` because that file contains types and functions that are used all over...someone developing Celeste needs to be very familiar with them. For all the other modules, I prefer `import`, because the functions in them aren't as second nature...it'd be easier to forget where and in what context these functions were defined